pages/index.js

```diff
@@ -99,36 +96,39 @@ export default function Home({ flowsDB }) {
     refreshData();
   }
 
-  function cancelEditFlow() {
-    setEditFormId(null);
-  }
+  const cancelEditFlow = () => setEditFormId(null);
 
-  function closeForm() {
-    setOpenForm(false);
-  }
+  const closeForm = () => setOpenForm(false);
+
+  const toggleOpenForm = () => setOpenForm((prev) => !prev);
+
+  const container = useRef(null);
+  useEffect(() => {
+    const instance = lottie.loadAnimation({
+      container: container.current,
+      renderer: "svg",
+      loop: true,
+      autoplay: true,
+      animationData: require("/public/animations/yoga-animation-start.json"),
+    });
+    return () => instance.destroy();
+  }, []);
 
   return (
     <div>
       <Head>
-        <title>My App</title>
+        <title>Your daily Flow</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main>
-        <StyledH2>NAMASTE,</StyledH2>
+        <StyledH2>NAMASTE</StyledH2>
         {session && <StyledH2>{session.user.name.toUpperCase()}</StyledH2>}
+
         <h3>Let&apos;s flow together</h3>
```

Comment:
Is the h3 tag a subheader of StyledH2? It seems it belongs together. So you might put the content of h3 into H2 and style it by <span>